packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js

```diff
@@ -22,20 +25,27 @@ export default /* @ngInject */ ($stateProvider) => {
         );
       },
 
+      trackingPrefix: () =>
+        'PublicCloud::pci::projects::project::storages::cold_archive::',
+
       userList: /* @ngInject */ (projectId, allUserList) =>
         allUserList.filter((user) => user?.s3Credentials?.length > 0),
 
-      allUserList: /* @ngInject */ (
-        projectId,
-        PciStoragesObjectStorageService,
-      ) =>
-        PciStoragesObjectStorageService.getAllS3Users(projectId).then((users) =>
-          PciStoragesObjectStorageService.mapUsersToCredentials(
-            projectId,
-            users,
-          ),
+      allUserList: /* @ngInject */ (projectId, PciStoragesColdArchiveService) =>
+        PciStoragesColdArchiveService.getAllS3Users(projectId).then((users) =>
+          PciStoragesColdArchiveService.mapUsersToCredentials(projectId, users),
         ),
 
+      isUserTabActive: /* @ngInject */ ($transition$, $state) => () =>
+        $state
+          .href($state.current.name, $transition$.params())
+          .includes('users'),
```

Comment:
**todo:** `includes` can be misleading if we create a new state like `usersList`.So better to check for the exact state name.